src/common/dlfcn_win32.cc

```diff
@@ -0,0 +1,57 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=8 sw=2 smarttab
+/*
+ * Ceph - scalable distributed file system
+ *
+ * Copyright (C) 2020 SUSE LINUX GmbH
```

Comment:
are you sure about this? in general, the copyright belongs to the entity which you work for.

Reply:
Yes, Suse is sponsoring this work. Thanks for checking though.